executor/write_test.go

```diff
@@ -799,7 +799,77 @@ func (s *testSuite) TestPartitionedTableReplace(c *C) {
 			PARTITION p2 VALUES LESS THAN (7),
 			PARTITION p3 VALUES LESS THAN (9));`
 	tk.MustExec(testSQL)
-	testSQL = `replace replace_test (c1) values (1),(2),(NULL);`
+
+	testPartitionTableReplace1(tk, c)
```

Comment:
The range `testPartitionTableReplace1`  covered is too small, from 1~7
So many partition is not covered in the following test.